datasets/turkish_product_reviews/turkish_product_reviews.py

```diff
@@ -0,0 +1,64 @@
+"""Turkish Product Reviews"""
+
+from __future__ import absolute_import, division, print_function
+
+import logging
+import os
+
+import datasets
+
+
+_CITATION = ""
+
+_DESCRIPTION = """
+Turkish Product Reviews.
+This repository contains 235.165 product reviews collected online. There are 220.284 positive, 14881 negative reviews.
+"""
+
+_URL = "https://github.com/fthbrmnby/turkish-text-data/raw/master/reviews.tar.gz"
+_FILES_PATHS = ["reviews.pos", "reviews.neg"]
+
+_HOMEPAGE = "https://github.com/fthbrmnby/turkish-text-data"
+
+
+class TurkishProductReviews(datasets.GeneratorBasedBuilder):
+    VERSION = datasets.Version("1.0.0")
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "sentence": datasets.Value("string"),
+                    "sentiment": datasets.ClassLabel(names=["positive", "negative"]),
```

Comment:
```suggestion
                    "sentiment": datasets.ClassLabel(names=["negative", "positive"]),
```

Usually we set the names in this order because often we want the positive label to be at index 1 and the negative label at index 0.